BUILDING THE REPO.md

```diff
@@ -0,0 +1,11 @@
+# HOW TO BUILD THE REPO
+In order to run the code ,we have to first build it .
+
+## Here are the steps to build the repo
+1. Make a new workspace named catkin_ws(it should be specifically catkin_ws otherwise it will give an error).
```

Comment:
Better to give the commands that are needed to make a new work space named catkin_ws so that any newbie can do it.